src/eval/index.php

```diff
@@ -1,9 +1,12 @@
 <?php
+	// set me to false to turn off sandboxing
+	$sandbox_me = true;
```

Comment:
Sure, that works.  My theoretical worry is that someone would put up an unsandboxed version on a vhost without realizing, but you're probably right that people who run locally likely want an unsandboxed version without need to edit source.